### DISCUSSION TITLE
Unique background color for every day in visible month

### DISCUSSION THREAD

**Discussion by Andi365** [gpbl/react-day-picker#1975] [Support]
Hi all,

I am looking for a way to render a unique background color for each day visible in the picker. 

![image](https://github.com/gpbl/react-day-picker/assets/43875035/af23751b-3c90-433e-9b62-bb544f35c826)

I have tried to paint an example above, where some days are different colors.

My usecase:
I want to visualize a unique month structure. ex: My november spans from the first monday in november to the sunday before the first monday in next month. So all the days in my custom november i want to highlight with a specific color. 

I am curious if this is possible? I have trouble getting the current month and year and hence computing what my custom start and end date of the month is.

All help is appreciated!

**Comment by gpbl** [comment]
Hi @Andi365 could you provide a screenshot of the design you are trying to build, and an example of have you tried to do in [a code sandbox](https://codesandbox.io/p/sandbox/react-day-picker-wfy7mq)? 

This would help to answer your question more in details, but definitely looks like it is possible to style each day as you want: https://react-day-picker.js.org/basics/styling#styling-modifiers
